src/radical/pilot/utils/component.py

```diff
@@ -497,7 +497,7 @@ def create(cfg, session):
         #
         from radical.pilot import worker       as rpw
         from radical.pilot import pmgr         as rppm
-        from radical.pilot import umgr         as rpum
+        from radical.pilot import tmgr         as rpum
```

Comment:
maybe `... as rptm`?

Reply:
I think `pmgr` for `pilot manager` and `tmgr` for `task manager` is just right :-D